frontend/apps/reader/modules/readerfont.lua

```diff
@@ -141,7 +141,7 @@ end
 function ReaderFont:onGesture() end
 
 function ReaderFont:registerKeyEvents()
-    if Device:hasKeyboard() then
+    if Device:hasKeyboard() and not Device:hasFiveWay() then
```

Comment:
What's this exception for? This doesn't look like something any Kindle keyboard owner will be happy with.